<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchAddSelectWindow</class>
 <widget class="QDialog" name="BatchAddSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BatchAddSelect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Curve1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Source1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="source1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="color1">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="style1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>SolidLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotDotLine</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thickness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="width1">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Curve2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Source2:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="source2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="color2">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="style2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>SolidLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotDotLine</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thickness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="width2">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Curve3</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Source3:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="source3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="color3">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="style3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>SolidLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotDotLine</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thickness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="width3">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Curve4</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Source4:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="source4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="color4">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>select</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="style4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>SolidLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotDotLine</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_16">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thickness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="width4">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>398</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="checkButton">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>22</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
